plugin/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go

```diff
@@ -37,7 +37,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
   "predicates": [
     {"name": "MatchNodeSelector"},
     {"name": "PodFitsResources"},
-    {"name": "PodFitsPorts"},
+    {"name": "PodFitsHostPorts"},
```

Comment:
if we want to rename it, that's fine, but the old name needs to stay registered as well. add a new item in the 1.1 test JSON below with any new predicate names